src/test/java/no/unit/nva/database/DatabaseServiceTest.java

```diff
@@ -40,68 +42,48 @@ public void databaseServiceShouldHaveAMethodForInsertingAUser() throws InvalidUs
 
     @Test
     public void databaseServiceShouldInsertValidItemInDatabase() throws InvalidUserException, InvalidRoleException {
-        UserDto insertedUser = createSampleUser();
+        UserDto insertedUser = createSampleUserAndAddUserToDb(SOME_USERNAME, SOME_INSTITUTION, SOME_ROLE);
```

Comment:
refactoring and grouping the common actions of creating a sample user and inserting the entry to the database.